GPU/Common/TextureCacheCommon.cpp

```diff
@@ -386,3 +443,35 @@ bool TextureCacheCommon::GetCurrentClutBuffer(GPUDebugBuffer &buffer) {
 	memcpy(buffer.GetData(), clutBufRaw_, 1024);
 	return true;
 }
+
+u32 TextureCacheCommon::EstimateTexMemoryUsage(const TexCacheEntry *entry) {
+	const u16 dim = entry->dim;
+	const u8 dimW = ((dim >> 0) & 0xf);
+	const u8 dimH = ((dim >> 8) & 0xf);
+
+	u32 pixelSize = 2;
+	switch (entry->format) {
+	case GE_TFMT_CLUT4:
+	case GE_TFMT_CLUT8:
+	case GE_TFMT_CLUT16:
+	case GE_TFMT_CLUT32:
+		// We assume cluts always point to 8888 for simplicity.
+		pixelSize = 4;
+		break;
+	case GE_TFMT_4444:
+	case GE_TFMT_5551:
+	case GE_TFMT_5650:
+		break;
+
+	case GE_TFMT_8888:
+	case GE_TFMT_DXT1:
+	case GE_TFMT_DXT3:
+	case GE_TFMT_DXT5:
+	default:
+		pixelSize = 4;
```

Comment:
I do realize this is just moved code, so no need to fix it now - but for the DXT formats, 4 is not a good estimate - 0.5 for DXT1 and 1 for DXT3 and DXT5. 

These formats aren't very common though as they perform badly on the PSP so probably hardly matters.